db/models.py

```diff
@@ -14,3 +14,35 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name='movies')
+    genres = models.ManyToManyField(Genre, related_name='movies')
+
+    def __str__(self) -> str:
+        return f"{self.title}"
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> str:
+        return f"{self.name}"
+
+    @property
+    def capacity(self):
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE, related_name="session")
```

Comment:
The related name 'session' for the `cinema_hall` ForeignKey might be misleading if a CinemaHall can have multiple sessions. Consider using a plural form like 'sessions' to better reflect the relationship.